src/main/java/org/opensearch/ad/AnomalyDetectorJobRunner.java

```diff
@@ -246,7 +246,7 @@ protected void runAdJob(
         String user = userInfo.getName();
         List<String> roles = userInfo.getRoles();
 
-        String resultIndex = jobParameter.getResultIndex();
+        String resultIndex = jobParameter.getCustomResultIndex();
```

Comment:
Would it always be a custom result index in this path? Or all result indexes now custom?

Reply:
yes. When this function returns null, we uses default result index.

Reply:
Makes sense.